examples/defaulter-gen/generators/defaulter.go

```diff
@@ -497,6 +498,24 @@ func mustEnforceDefault(t *types.Type, depth int, omitEmpty bool) (interface{},
 	}
 }
 
+var refRE = regexp.MustCompile(`^ref\((("(?P<pkg>.*)")\.)?(?P<ident>[^"]*)\)$`)
+var refREPkgIndex = refRE.SubexpIndex("pkg")
+var refREIdentIndex = refRE.SubexpIndex("ident")
+
+// parseAsRef looks for strings that match one of the following:
+//   - ref(Ident)
+//   - ref("pkgpath".Ident)
```

Comment:
the doc and regex use `DQ "` around the package path but the example code above doesn't? Which is it?